armi/cli/checkInputs.py

```diff
@@ -36,6 +38,31 @@ def _runInspectorOnSettings(cs):
     return issues
 
 
+class ExpandBlueprints(EntryPoint):
+    """
+    Perform expansion of !include directives in a blueprint file.
```

Comment:
I'd add more to this for more context to a new user. Maybe:

`Process any !include directives in a blueprint file and print the expanded content to stdout`